test/test_proxy_tensor.py

```diff
@@ -40,9 +40,9 @@ def strip_end(s, suffix):
 
 def show_guards(gm):
     names = [strip_end(n, "_1") for n in fx_placeholder_targets(gm)]
-    return "\n".join(
-        gm.shape_env.produce_guards(fx_placeholder_vals(gm), names, _simplified=True, constraint_inputs=None)
-    )
+    guards = gm.shape_env.produce_guards_and_constraints(fx_placeholder_vals(gm),
+                                                         names, _simplified=True, constraint_inputs=None)[0]
```

Comment:
funny formatting

Reply:
Yeah, I can change it to other format. lintrunner -a somehow didn't work.